demos/guru_scripts/docker/tutorial/4.x/script/load.gsql

```diff
@@ -1,16 +1,20 @@
 USE GRAPH financialGraph
 
 DROP JOB load_data
+DROP DATA_SOURCE s3_data_source_0
```

Comment:
can you add a section to make s3? btw: is the secret.key public?

Reply:
Ok, I didn't use valid secret.key and will not leak the key to user.